packages/vuetify/src/components/VDatePicker/VDatePickerMonth.tsx

```diff
@@ -39,7 +48,14 @@ export const makeVDatePickerMonthProps = propsFactory({
     type: String,
     default: 'picker-reverse-transition',
   },
-
+  events: {
+    type: [Array, Function, Object] as PropType<DatePickerEvents | null>,
+    default: () => null,
+  },
+  eventColor: {
+    type: [Array, Function, Object, String] as PropType<DatePickerEventColors>,
+    default: () => 'warning',
```

Comment:
will probably default this to null and let the default color be a surface shade, like surface-variant.